demo/README.rst

```diff
@@ -0,0 +1,38 @@
+Getting Started
+===============
+
+This folder contains tutorials on how to interface with SLRealizer to generate mock catalogs and use the catalogs to do interesting analyses.
+ 
+.. list-table::
+   :widths: 10 20 10 10
+   :header-rows: 1
+
+   * - Notebook
+     - Short description
+     - Links
+     - Owner
+
+
+   * - **Example SLRealizer Usage**
+     - Tutorial on generating mock LSST DRP Source and Object catalogs from OM10 or the SDSS data
+     - `ipynb <Example+SLRealizer+Usage.ipynb>`_
+     - `Ji Won Park <https://github.com/jiwoncpark/SLRealizer/issues>`_
+
+   * - **Comparing OM10 and SDSS Objects**
+     - Comparison of realized lenses (from OM10) and non-lenses (from SDSS)
+     - `ipynb <Comparing+OM10+vs+SDSS+Objects.ipynb>`_,
+       `rendered <https://nbviewer.jupyter.org/github/LSSTScienceCollaborations/StackClub/blob/rendered/GettingStarted/HelloWorld.nbconvert.ipynb>`_
```

Comment:
This line should not be here! We don't (yet) have rendered notebooks to point at...

Reply:
Removed!